Pima indians diabetes and linear classifier/Conclusion/task-info.yaml

```diff
@@ -1,32 +1,160 @@
 type: theory
 files:
-  - name: __init__.py
-    visible: false
-    propagatable: false
   - name: pima-indians-diabetes.csv
     visible: true
-  - name: task.py
-    visible: true
-  - name: loss_functions.py
+  - name: stochastic_gradient_descent.py
     visible: true
+    placeholders:
+      - offset: 562
+        length: 193
+        placeholder_text: "# TODO"
+        dependency:
+          lesson: Pima indians diabetes and linear classifier
+          task: Stochastic Gradient Descent
+          file: stochastic_gradient_descent.py
+          placeholder: 1
+        is_visible: false
+      - offset: 840
+        length: 16
+        placeholder_text: "# TODO"
+        dependency:
+          lesson: Pima indians diabetes and linear classifier
+          task: Stochastic Gradient Descent
+          file: stochastic_gradient_descent.py
+          placeholder: 2
+        is_visible: false
+      - offset: 905
+        length: 292
+        placeholder_text: "# TODO"
+        dependency:
+          lesson: Pima indians diabetes and linear classifier
+          task: Stochastic Gradient Descent
+          file: stochastic_gradient_descent.py
+          placeholder: 3
+        is_visible: false
+      - offset: 1214
+        length: 6
+        placeholder_text: "# cost values per iteration"
+        dependency:
+          lesson: Pima indians diabetes and linear classifier
+          task: Stochastic Gradient Descent
+          file: stochastic_gradient_descent.py
+          placeholder: 4
+        is_visible: false
   - name: gradient_descent.py
     visible: true
-  - name: precision_recall.py
-    visible: true
-  - name: stochastic_gradient_descent.py
+    placeholders:
+      - offset: 300
+        length: 553
+        placeholder_text: "# TODO: run gradient descent, update weights until convergence"
```

Comment:
placeholder_text: "# TODO: Run gradient descent and update weights until convergence"